e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -460,6 +460,36 @@ test.describe('Example Imagery in Display Layout', () => {
 
 test.describe('Example Imagery in Flexible layout', () => {
   let flexibleLayout;
+  test.describe('Example Imagery in Flexible layout', () => {
```

Comment:
Let's remove this nested describe as its unnecessary

Reply:
Sorry, forgot to add the last commit.